Assets/_Developers/Rolo/Scripts/MapRework/MapStationDrag.cs

```diff
@@ -0,0 +1,101 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class MapStationDrag : MonoBehaviour
+{
+    [SerializeField] private LayerMask RaycastLayers = default(LayerMask);
+
+    Transform stationTransform;
+
+    bool isPressed;
+    // Start is called before the first frame update
+    void Start()
+    {
+        stationTransform = null;
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (isPressed)
```

Comment:
Nesting can be avoided here.

At the top of update, you can write if (stationTransform == null) return; 

and then only check for isPressed